WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -413,11 +431,12 @@ decisions = {
 			OR = {
 				religion_group = christian
 				religion_group = pagan_group
+				has_romance_culture_trigger = yes
+				culture = roman
 			}
 			age = 16
 			prisoner = no
 			NOT = { trait = incapable }
-			has_romance_culture_trigger = yes
```

Comment:
In order to be able to restore, unite or claim any of the Roman empires in the mod, you need to be of a Romance culture. This is to reflect why characters such as Theoderic and Charlemagne couldn't and didn't choose to recreate it. Charlemagne instead created a Carolingian empire, for which there will be flavour and content with alternate paths for other dynasties gaining the imperial dignity. This also ties into the melting-pot transition of moving away from the Roman heritage. 

It is good that you are curious and question things, but outright removing parts of the mod is a bit unnecessary and fairly redundant. The code I add isn't made in a certain way "because", there is always a reasoning behind it.